docs/04-reference/01-cli-reference/strm/approve/index.md

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
Did you add the cli reference files by hand? They are auto-generated from the cli project, so they shouldn't be changed manually. Please revert them.